capstone/src/main/webapp/script.js

```diff
@@ -20,11 +20,14 @@ async function getData() {
     let pollingInfo = await lookupPollingPlace(address);
     console.log('Looked up polling place: ' + JSON.stringify(pollingInfo));
     
+    //putting all address on DOM
```

Comment:
With comments, always put a space between the "//" and the start of the comment. If it's a complete sentence (and kind of is) use a capital at the start and a period at the end.

However, in this case I would delete this comment and rename the function "addAddressesToDom" so the comment isn't necessary.